files/static/js/genfors.js

```diff
@@ -123,40 +129,55 @@ Genfors = (function () {
                         for (var x = 0; x < options.length; x++) {
                             var key = options[x];
                             var value = r[key];
-                            var percent = Genfors.get_percent(value, current_votes);
-                            html += '<strong>' + key + '</strong>: ' + value + ' stemme';
-                            if (value != 1) {
-                                html += 'r';
+                            var percent;
+                            if (key == 'Blankt') {
+                                percent = Genfors.get_percent(value, current_votes);
                             }
-                            type = "";
-                            if (key == 'Ja') {
-                                type = 'success';
+                            else{
+                                percent = Genfors.get_percent(value, votes_for_alternative);
                             }
-                            else if (key == 'Nei') {
-                                type = 'danger';
+                            html += '<strong>' + key + '</strong>: ' + value + ' stemme' + ((value>1)?'r':'');
+                            type = "";
+                            if (key == 'Blankt') { 
+                                type = 'warning';
+                                html += '<p /><div class="progress"><div class="progress-bar progress-bar-' + type + '" role="progressbar" aria-valuenow="' + percent + 
```

Comment:
Empty `<p>`-tags should not be used. If you're looking for blank lines, use CSS (margin) or `<br />`.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p#Notes